x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -367,23 +370,35 @@ export class ESSearchSource extends AbstractESSource {
           defaultMessage: `Results limited to most recent {topHitsSize} documents per entity.`,
           values: { topHitsSize: this._descriptor.topHitsSize }
         });
-        return `${entitiesFoundMsg} ${trimmedMsg}`;
+        return {
+          tooltipContent: `${entitiesFoundMsg} ${trimmedMsg}`,
+          areResultsTrimmed: false
+        };
       }
 
-      return entitiesFoundMsg;
+      return {
+        tooltipContent: entitiesFoundMsg,
+        areResultsTrimmed: false
+      };
     }
 
     if (meta.areResultsTrimmed) {
-      return i18n.translate('xpack.maps.esSearch.resultsTrimmedMsg', {
-        defaultMessage: `Results limited to first {count} documents.`,
-        values: { count: featureCollection.features.length }
-      });
+      return {
+        tooltipContent: i18n.translate('xpack.maps.esSearch.resultsTrimmedMsg', {
+          defaultMessage: `Results limited to first {count} documents.`,
```

Comment:
Can you add the iInCircle icon to the beginning of this line so that users can correlate why there's an icon?